schemas/schema_helpers.go

```diff
@@ -178,6 +178,33 @@ func diffSuppressInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc
 	}
 }
 
+// diffSuppressAppInterfaceListOrder checks whether we have the newly created AppInterface
+// or not. It suppress any differences after an AppInstance is created.
+// AppInterfaces are never updated after they are created
+func diffSuppressAppInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc {
+	return func(key, oldValue, newValue string, d *schema.ResourceData) bool {
+		oldData, newData := d.GetChange(mapKey)
+		if newData == nil && oldData == nil {
+			return true
+		}
+
+		if oldData == nil {
+			return false
+		}
+		if newData == nil {
+			return false
+		}
+
+		o := oldData.([]interface{})
+		n := newData.([]interface{})
+		if len(o) != len(n) {
```

Comment:
So if they delete one interface and add another the change would be ignored. If they added or deleted one, the change would be reported but the backend would return an error, correct?

Reply:
correct, and i'll fix this